drivers/performance_measurement/app/models/performance_measurement/report.rb

```diff
@@ -1196,21 +1227,27 @@ def spm_fields
           history_source: :m7b_history,
           questions: [
             {
-              name: :moved_in_destination, # NOTE: destination 0 == stayer in the SPM
-              value_calculation: destination_calculation,
+              name: :moved_in_destination,
+              value_calculation: default_calculation,
             },
           ],
           client_project_rows: [
             ->(spm_enrollment) {
-              return unless destination_calculation.call(spm_enrollment)
```

Comment:
I think 7b.2 universe is only enrollments with a move-in, so this check isn't needed

Reply:
I'm not quite sure I follow, my mental model is that there's an implied filter on the lamba so that it only runs for enrollments who would fall into the universe defined in cell (7b.1 C2). However I'll add logic to check for stayers here in addition to destination